### PR TITLE
Removed Milan eu-south-1 , Secretmanager

### PR DESCRIPTION
**Pull request by shajeena**

### CODE REVIEW

**Review by vaibhavkulkarniibm — COMMENTED** [review]
Since we need secretsmanager permissions for creating secret with OCP and MAS Credentials, we need to have `"secretsmanager:CreateSecret"` but not below permissions. 
```
"secretsmanager:GetResourcePolicy",
"secretsmanager:GetSecretValue",
"secretsmanager:DescribeSecret",
"secretsmanager:ListSecretVersionIds",
"secretsmanager:ListSecrets"
```
Did you observe an error in bootnode logs due to lack of these permissions? 
Also, there seems some indent issue.

**Comment by shajeena** [comment]
> Since we need secretsmanager permissions for creating secret with OCP and MAS Credentials, we need to have `"secretsmanager:CreateSecret"` but not below permissions.
> 
> ```
> "secretsmanager:GetResourcePolicy",
> "secretsmanager:GetSecretValue",
> "secretsmanager:DescribeSecret",
> "secretsmanager:ListSecretVersionIds",
> "secretsmanager:ListSecrets"
> ```
> 
> Did you observe an error in bootnode logs due to lack of these permissions? Also, there seems some indent issue.

@vaibhavkulkarniibm  - I have added CreateSecret ,GetSecretValue  removed others .  I saw the below error in logs , so kept GetSecretValue .

`An error occurred (ResourceNotFoundException) when calling the GetSecretValue operation: Secrets Manager can't find the specified secret.`

**Comment by shajeena** [comment]
removed the getsecretvalue

File: aws/master-cft/byol-upi/cft-mas-core-dev.json [hunk 1]
```diff
@@ -457,7 +454,8 @@
                                         "logs:CreateLogGroup",
                                         "logs:CreateLogStream",
                                         "logs:PutLogEvents",
-                                        "logs:DescribeLogStreams"
+                                        "logs:DescribeLogStreams",
+                                        "secretsmanager:CreateSecret"
```
Comment (vaibhavkulkarniibm) [inline aws/master-cft/byol-upi/cft-mas-core-dev.json:458]: This `secretsmanager:CreateSecret `correct as its boot node policy.

File: aws/master-cft/byol-upi/cft-mas-core-dev.json [hunk 2]
```diff
@@ -540,7 +538,8 @@
                                         "logs:CreateLogGroup",
                                         "route53:GetHostedZone",
                                         "ec2:DescribeVpcs",
-                                        "ec2:DescribeSubnets"
+                                        "ec2:DescribeSubnets",
+                                        "secretsmanager:CreateSecret"
```
Comment (vaibhavkulkarniibm) [inline aws/master-cft/byol-upi/cft-mas-core-dev.json:542]: This `secretsmanager:CreateSecret ` is not needed, It's for LambdaRole

File: aws/master-cft/byol-upi/cft-mas-core.json [hunk 3]
```diff
@@ -540,7 +538,8 @@
                                         "logs:CreateLogGroup",
                                         "route53:GetHostedZone",
                                         "ec2:DescribeVpcs",
-                                        "ec2:DescribeSubnets"
+                                        "ec2:DescribeSubnets",
+                                        "secretsmanager:CreateSecret"
```
Comment (vaibhavkulkarniibm) [inline aws/master-cft/byol-upi/cft-mas-core.json:542]: This `secretsmanager:CreateSecret ` is not needed, It's for LambdaRole

File: aws/master-cft/byol-upi/cft-mas-core.json [hunk 4]
```diff
@@ -457,7 +454,8 @@
                                         "logs:CreateLogGroup",
                                         "logs:CreateLogStream",
                                         "logs:PutLogEvents",
-                                        "logs:DescribeLogStreams"
+                                        "logs:DescribeLogStreams",
+                                        "secretsmanager:CreateSecret"
```
Comment (vaibhavkulkarniibm) [inline aws/master-cft/byol-upi/cft-mas-core.json:458]: This `secretsmanager:CreateSecret `is correct as its boot node policy.

**Comment by vaibhavkulkarniibm** [comment]
Looks good.

**Pull request merged**